package-lock.json

```diff
@@ -2044,8 +2045,9 @@
     },
     "batch-processor": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/batch-processor/-/batch-processor-1.0.0.tgz",
-      "integrity": "sha1-dclcMrdI4IUNEMKxaPa9vpiRrOg="
+      "resolved": "https://artifactory.ciena.com/api/npm/blueplanet-npm/batch-processor/-/batch-processor-1.0.0.tgz",
```

Comment:
These are resolving to the incorrect registry.  Read the **".npmrc file (05 April 2018)"** section of the https://confluence.ciena.com/display/frost/Changes+to+Ember+Addons+and+Apps document on for how to resolve.